### PR TITLE
Fix: polish code base

### PR DESCRIPTION
**Pull request by BryanElmer**

### CODE REVIEW

**Review by zonglun99 — APPROVED** [review]

File: src/main/java/seedu/commands/RemoveCommand.java [hunk 1]
```diff
@@ -92,4 +93,5 @@ private void updateShopList(Ui ui, ShoppingList shoppingList, Item itemRemoved,
             ui.printShopUpdateMessage(addedItem, qtyInShop);
         }
     }
+    //@@author BryanElmer
```
Comment (uosjapuelks) [inline src/main/java/seedu/commands/RemoveCommand.java:96]: When closing you authorship claim, use `//@@author` without your github username after. Also, if you're going to claim all the way to the bottom, you don't have to end with `//@@author`.

File: src/main/java/seedu/commands/ResetCommand.java [hunk 2]
```diff
@@ -34,3 +35,4 @@ public void execute(Ui ui, Parser parser, ItemList itemList, ShoppingList shoppi
         }
     }
 }
+//@@author BryanElmer
```
Comment (uosjapuelks) [inline src/main/java/seedu/commands/ResetCommand.java:38]: Same here.

File: src/main/java/seedu/commands/ShopListCommand.java [hunk 3]
```diff
@@ -32,3 +33,4 @@ public void execute(Ui ui, Parser parser, ItemList itemList, ShoppingList shoppi
         ui.printListMessage(listOfItems, "", true);
     }
 }
+//@@author BryanElmer
```
Comment (uosjapuelks) [inline src/main/java/seedu/commands/ShopListCommand.java:36]: And Here.

File: src/main/java/seedu/commands/ShopResetCommand.java [hunk 4]
```diff
@@ -35,3 +36,4 @@ public void execute(Ui ui, Parser parser, ItemList itemList, ShoppingList shoppi
         }
     }
 }
+//@@author BryanElmer
```
Comment (uosjapuelks) [inline src/main/java/seedu/commands/ShopResetCommand.java:39]: here too.

File: src/main/java/seedu/commands/UpdateCommand.java [hunk 5]
```diff
@@ -68,4 +69,5 @@ private void updateShopList(Ui ui, ShoppingList shoppingList, Item updatedItem,
             ui.printShopUpdateMessage(addedItem, qtyInShop);
         }
     }
+    //@@author BryanElmer
```
Comment (uosjapuelks) [inline src/main/java/seedu/commands/UpdateCommand.java:72]: This too.

File: src/main/java/seedu/storage/Storage.java [hunk 6]
```diff
@@ -188,8 +191,9 @@ private void readFromLogFile(File logFile) throws FileNotFoundException {
             addSavedNotification(logScanner.nextLine());
         }
     }
-    //@@author BryanElmer
+    //@@author zonglun99
```
Comment (uosjapuelks) [inline src/main/java/seedu/storage/Storage.java:195]: What's going on here?

File: src/test/java/seedu/storage/StorageTest.java [hunk 7]
```diff
@@ -55,3 +56,4 @@ void addSavedShopItem_correctSavedShopItemInput_expectCorrectFormatInShopList()
         }
     }
 }
+//@author BryanElmer
```
Comment (uosjapuelks) [inline src/test/java/seedu/storage/StorageTest.java:59]: This too.

**Pull request merged**